docs/APISocketOptions.md

```diff
@@ -1094,33 +1099,41 @@ The value set here will be effectively aligned to the multiple of payload size.
 
 | OptName           | Since | Binding | Type       |  Units  |   Default  | Range  | Dir | Entity |
 | ----------------- | ----- | ------- | ---------- | ------- | ---------- | ------ | --- | ------ |
-| `SRTO_RCVLATENCY` | 1.3.0 | pre     | `int32_t`  | msec    | *          | 0..    | RW  | GSD    |
+| `SRTO_RCVLATENCY` | 1.3.0 | pre     | `int32_t`  | ms      | *          | 0..    | RW  | GSD    |
 
-- Latency value in the receiving direction. This value is only significant when
-`SRTO_TSBPDMODE` is set to true.
+- The latency value in the receiving direction of the current SRT connection.
+This value is only significant when [`SRTO_TSBPDMODE`](#SRTO_TSBPDMODE) is enabled.
 
-- Latency refers to the time that elapses from the moment a packet is sent 
-to the moment when it's delivered to a receiver application. The SRT latency 
-setting should be a buffer large enough to cover the time spent for sending, 
-unexpectedly extended RTT time, and the time needed to retransmit any
-lost UDP packet. The effective latency value will be the maximum between the 
-`SRTO_RCVLATENCY` value and the value of `SRTO_PEERLATENCY` set by 
-the peer side. **This option in pre-1.3.0 version is available only as** 
-`SRTO_LATENCY`. Note that the real latency value may be slightly different 
-than this setting due to the impossibility of perfectly measuring exactly the 
-same point in time at both parties simultaneously. What is important with 
-latency is that its actual value, once set with the connection, is kept constant 
-throughout the duration of a connection.
+- **Default value**: 120 ms (depicted as 0) in Live mode, 0 in File mode (see [`SRTO_TRANSTYPE`](#SRTO_TRANSTYPE)).
 
-- Default value: 120 in Live mode, 0 in File mode (see [`SRTO_TRANSTYPE`](#SRTO_TRANSTYPE)).
+- The latency value defines the **minimum** receiver buffering delay before delivering an SRT data packet
```

Comment:
The use of `minimum` is kinda unfortunate here. This is minimum from the perspective of negotiation of the effective value, not from the perspective of applying this value to the received data, while it sounds here as the latter.

You can also check this in the code, but as far as I remember how it works is that the transmission start time latch happens at the moment of receiving the first data packet. The time when it was received for that very packet is simply treated as the expected reception time, and the time from the timestamp is then used to define the delta used to calculate the value of the expected reception time for the next packets. Then expected reception time + latency + dfift = play time. I might describe it also as:

First packet:
```
dTp = [packet's timestamp];
Texp = [now];  <-- here we state that expected arrival time is when it was arrived
Tstart = Texp - dTp;  <-- here we define Tstart the very first time
Tplay = Texp + [latency] + [drift];
```
Next packets:
```
dTp = [packet's timestamp];
dTs = [collected time from over-32bit time overflows];
Texp = Tstart + dTs + dTp;  <-- here we can use Tstart we defined previously
Tplay = Texp + [latency] + [drift];
```

Reply:
> The use of minimum is kinda unfortunate here. This is minimum from the perspective of negotiation of the effective value, not from the perspective of applying this value to the received data, while it sounds here as the latter.

The word "minimum" is used here for the reading delay (TSBPD), as the actual delay can be bigger than the one set on the socket.

> The time when it was received for that very packet is simply treated as the expected reception time, and the time from the timestamp is then used to define the delta used to calculate the value of the expected reception time for the next packets. Then expected reception time + latency + dfift = play time. I might describe it also as:

From the sender's and the whole transmission point of view, "the time when it was received" gives additional RTT/2 to the end-to-end latency, as described.